src/pages/_app.tsx

```diff
@@ -25,7 +26,13 @@ const App = ({ Component, pageProps }: AppPropsWithLayout) => {
         `}
       </style>
       <ChakraProvider theme={theme}>
-        {getLayout(<Component {...pageProps} />)}
+        <RootLayout
+          contentIsOutdated={pageProps.contentIsOutdated}
```

Comment:
I think this `contentIsOutdated` doesn't exist as a page prop. It is part of the `frontmatter`.
```suggestion
          contentIsOutdated={pageProps.frontmatter.contentIsOutdated}
```

or maybe we could just copypaste what we were doing before

```ts
const { frontmatter, lastDeployDate, contentNotTranslated } = pageProps

const rootLayoutProps = {
  contentIsOutdated: frontmatter.isOutdated,
  contentNotTranslated,
  lastDeployDate,
}

// and then just
<RootLayout {...rootLayoutProps}>
```